### ISSUE TITLE
scylla_setup RAID device format unclear

### ISSUE THREAD

**Issue by pveentjer** [scylladb/scylla#8173]
*Installation details*
Scylla version (or git commit hash): 4.3
OS: Centos

It isn't clear what the format should be to pass the devices for the RAID setup. Should it be comma separated for example?

```
Are you sure you want to setup RAID0 and XFS?
If you choose Yes, the selected drive will be reformated, erasing all existing data in the process.
[YES/no]y
Please select unmounted disks from the following list: /dev/xvdb, /dev/nvme0n1, /dev/nvme2n1, /dev/nvme1n1, /dev/nvme3n1
type 'cancel' to cancel RAID/XFS setup.
type 'done' to finish selection. Selected: 
> /dev/nvme0n1, /dev/nvme2n1, /dev/nvme1n1, /dev/nvme3n1
 /dev/nvme2n1 not found
 /dev/nvme1n1 not found
 /dev/nvme3n1 not found
```

A simple addition like 'comma separated list of devices without spaces' e.g. would be more than sufficient.

**Comment by slivne** [comment]
you select a disk in each iteration and then enter "done" to mark the end.

**Comment by syuu1228** [comment]
Fixed by 870c3a28c1a06decfbd80809218b2c16721c7fa2
